library/src/main/java/com/novoda/downloadmanager/StorageRequirementRuleFactory.java

```diff
@@ -8,11 +8,30 @@ private StorageRequirementRuleFactory() {
         // Uses static factory methods.
     }
 
+    /**
+     * Creates a storage requirement rule, where the storage bytes must be greater than the
+     * given bytes after a download completes. e.g. if specifying 100mb then the rule will
```

Comment:
should be 100MB (b -> bits, B -> bytes)

Reply:
Nit: `completes. e.g. if` -> `completes. E.g., if` 💃